_episodes/l2-02-remote_repositories.md

```diff
@@ -41,6 +37,8 @@ keypoints:
 - A local branch synchronised with a remote one is said to be **tracking** that
   remote branch.
 - The remote branch being tracked is called the **upstream branch**.
+- As mentioned, **we will be working in this course with the main branch only**, but the
```

Comment:
Where is this previously mentioned? In a previous episode?

Reply:
Well... I should have said that this should be mentioned in the previous episode, commits and history, so a small task for @dc2917 .